styles.css

```diff
@@ -753,3 +775,660 @@ input[type="email"]:focus:invalid,
+    align-items: center;
+    padding: 16px;
+    position: static;
+    width: 95%;
+    height: 552px;
+    left: calc(50% - 327px / 2);
+    top: 114px;
+    background: #fff;
+    box-sizing: border-box;
+    border-radius: 16px;
+    flex: none;
+    margin: 88px 2px;
+  }
+
+  .photo {
+    position: static;
+    width: 295px;
+    height: 220px;
+    left: 16px;
+    top: 16px;
+    margin-bottom: 6px;
+    background: url(photos/blog_drbl_4x.png);
+    border-radius: 8px;
+    flex: none;
+    display: flex;
+  }
+
+  .contact-me input[type="submit"]:hover {
+    background: #6070ff;
+    color: #fff;
+    border: 1px solid #000;
+    box-shadow: 0.001px 8px 16px rgba(64, 83, 252, 0.24);
+    border-radius: 8px;
+  }
+
+  .contact-me input[type="submit"]:active {
+    background: #2230d2;
+    color: #fff;
+    border: 1px solid #000;
+    box-shadow: 0.001px 8px 16px rgba(64, 83, 252, 0.24);
+    border-radius: 8px;
+  }
+}
+
+@media (min-width: 375px) {
+  body {
+    width: 100%;
+    overflow: hidden;
+  }
+
+  header {
+    height: 20px;
+    display: flex;
+    width: 90%;
+    justify-content: space-between;
+    padding-right: 50px;
+    font-size: 48px;
+    line-height: 60px;
+    align-items: center;
+    letter-spacing: 0.37px;
+    color: #091e42;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin: 12px 100px;
+  }
+
+  .about {
+    background-image: url(photos/Headerbg.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    flex-direction: column;
+    margin-top: 10px;
+    padding-left: 240px;
+    padding-right: 260px;
+    box-sizing: border-box;
+    border-radius: 1px 1px 1px 50px;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    background: white;
+    margin-top: 10px;
+    width: 100%;
+    padding: 24px;
+    box-sizing: border-box;
+    border-radius: 1px 1px 1px 50px;
+  }
+
+  .box2 {
+    width: 100%;
+    height: 100%;
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: bolder;
+    line-height: 52px;
+    display: flex;
+    color: #172b4d;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin: 12px 0;
+    flex-wrap: wrap;
+    font-size: 40px;
+    margin-top: 70px;
+  }
+
+  .wrapper {
+    display: grid;
+    width: 100%;
+    grid-template-rows: repeat(4, 596px) 114px;
+    row-gap: 114px;
+    height: 4350px;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 16px;
+    position: static;
+    width: 1156px;
+    height: 552px;
+    left: calc(50% - 327px / 2);
+    top: 114px;
+    background: #fff;
+    box-sizing: border-box;
+    border-radius: 16px;
+    flex: none;
+    margin-left: 207px;
+  }
+
+  .photo_1 {
+    position: static;
+    width: 50%;
+    height: 476px;
+    left: 16px;
+    top: 16px;
+    margin-bottom: 6px;
+    background: url(photos/blog_drbl_4x.png);
+    border-radius: 8px;
+    flex: none;
+    display: flex;
+    padding: 24px;
+  }
```

Comment:
- [ ] Great implementation on dividing space between image and description however our image is only occupying 50% of the existing space

Current implementation | Figma Design 
:-----------:|:-----------:
![image](https://user-images.githubusercontent.com/87689547/153888433-40dfac3b-9a5e-47bb-8fb7-7f36e2f529e1.png) | ![image](https://user-images.githubusercontent.com/87689547/153888536-802d8271-03c7-49d5-992a-bd8229b948ca.png)

Reply:
@Amrhub Thank you for your review. 
I am not sure about this screen if it exists already in my project. There is a screenshot of it.
![image](https://user-images.githubusercontent.com/58164905/153910225-952c9cf8-12dd-4a42-a604-0dd351e625b8.png)